scanpy/tests/external/test_scanorama_integrate.py

```diff
@@ -0,0 +1,18 @@
+import scanpy as sc
+import scanpy.external as sce
+
+
+def test_scanorama_integrate():
+    """
+    Test that Scanorama integration works.
+
+    This is a very simple test that just checks to see if the Scanorama
+    integrate wrapper succesfully added a new field to ``adata.obsm``
+    and makes sure it has the same dimensions as the original PCA table.
+    """
+    adata = sc.datasets.pbmc3k()
```

Comment:
Taking the `pbmc68k_reduced()` dataset, here, would hopefully speed up tests. Don't you think @ivirshup?

We'll run into infinite test times at some point if every tool takes several seconds to run.

Reply:
Good point. At the moment I've been more concerned about the install time of all the dependencies of external modules, but it's good to cut down everywhere we can.

Reply:
Made the changes accordingly!